permission_handler_apple/ios/Classes/PermissionManager.m

```diff
@@ -86,12 +86,6 @@ + (void)openAppSettings:(FlutterResult)result {
                                  completionHandler:^(BOOL success) {
                                      result([[NSNumber alloc] initWithBool:success]);
                                  }];
-    } else if (@available(iOS 8.0, *)) {
```

Comment:
This logic never hit `-[UIApplication openURL:]` on iOS 9.x, though it probably should have. 6a3bc62